middle_end/flambda2/from_lambda/lambda_to_flambda.ml

```diff
@@ -854,12 +892,14 @@ let rec cps acc env ccenv (lam : L.lambda) (k : cps_continuation)
               k_exn)
           k_exn)
       k_exn
-  | Ltrywith (body, id, _duid, handler, kind) ->
-    (* CR sspies: dropping [debug_uid]; address in subsequent PR. *)
+  | Ltrywith (body, id, duid, handler, kind) ->
     let dbg = Debuginfo.none (* CR mshinwell: fix [Lambda] *) in
     let body_result = Ident.create_local "body_result" in
+    let body_result_lambda_duid = L.debug_uid_none in
     let region = Ident.create_local "try_region" in
+    let region_duid = Flambda_debug_uid.none in
```

Comment:
It would be kind of fun to be able to see in the debugger that register %rbx or whatever contains the address of the local stack pointer at foo.ml:42 in the code...